<launch>
   
   <node pkg="rflysim_sensor_rospkg" type="image_receive.py" name="rgb_receive" output="screen">
     <param name="port" value="9999" />
     <!-- IP地址为NX/接收端的局域网IP -->
     <param name="IP" value="192.168.10.116" />
     <param name="pub_topic_name" value="/rgb_image_topic" />
     <param name="type" value="rgb"/>
     <param name="frame_id" value="camera_color_optical_frame"/>
     <param name="encoding" value="bgr8"/>
   </node>

</launch>
